source/common/filter/tcp_proxy.cc

```diff
@@ -9,20 +9,83 @@
 
 #include "common/common/assert.h"
 #include "common/json/config_schemas.h"
+#include "common/common/empty_string.h"
 #include "common/json/json_loader.h"
 
 namespace Filter {
 
+TcpProxyConfig::Route::Route(const Json::Object& config) {
+  if (config.hasObject("cluster")) {
+    cluster_name_ = config.getString("cluster");
+  } else {
+    throw EnvoyException("tcp proxy: route without cluster");
```

Comment:
With schema support now, can this actually happen? I don't think it can, so should probably delete.